src/Illuminate/Http/Client/Factory.php

```diff
@@ -250,7 +250,7 @@ public function assertSequencesAreEmpty()
      * @param  callable  $callback
      * @return \Illuminate\Support\Collection
      */
-    public function recorded($callback)
+    public function recorded($callback = null)
```

Comment:
This is a breaking change to the method signature. Best that this is sent to master.